velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1303,14 +1303,17 @@ struct DateParseFunction {
     }
 
     auto dateTimeResult =
-        format_->parse(std::string_view(input.data(), input.size()), true)
-            .value();
+        format_->parse(std::string_view(input.data(), input.size()));
```

Comment:
nit: can't you do implicit cast from StringView into string_view? If not, explicit should work:
```
  format_->parse(input);
```
or
```
  format_->parse((std::string_view)input);
```